SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -593,6 +593,12 @@ int64 USpatialActorChannel::ReplicateActor()
 		}
 	}
 
+	if (Actor->GetIsHiddenDirty())
```

Comment:
@samiwh what's the context of the question?
@eviK88 I think the answer is yes it can be changed more than once depending on game code correct? However once marked dirty it's dirty regardless

Reply:
Yep it depends on the game code.